src/hotspot/share/services/attachListener.cpp

```diff
@@ -365,6 +403,7 @@ static AttachOperationFunctionInfo funcs[] = {
   { "setflag",          set_flag },
   { "printflag",        print_flag },
   { "jcmd",             jcmd },
+  { "getVersion",       get_version },
```

Comment:
It's a bit of a nit, but "dumpheap" and other existing commands never use caps but the new "getVersion" does?

Reply:
We have "agentProperties" command, but this is the only exception.
Renamed to "getversion".